### PR TITLE
fix crash when deleting more than 21 vouchers

### PR DESCRIPTION
**Pull request by SektorDV**
I want to merge this change because it resolves the issue of the dashboard crashing when more than 21 vouchers are selected for deletion.

### Pull Request Checklist

1. [ ] This code contains UI changes
2. [ ] All visible strings are translated with proper context including data-formatting
3. [ ] Attributes `[data-test-id]` are added for new elements
4. [ ] Changes are mentioned in the changelog
5. [ ] The changes are tested in different browsers and in light/dark mode

### Test environment config

API_URI=https://master.staging.saleor.cloud/graphql/

### CODE REVIEW

**Comment by patrys** [comment]
Task linked: [SALEOR-3967 Dashboard error on removing multiple vouchers](https://app.clickup.com/t/2549495/SALEOR-3967)

**Review by orzechdev — APPROVED** [review]
Maybe it's also worth to see if there are similar places in dashboard where same situation may occur, besides vouchers?

File: src/discounts/index.tsx [hunk 1]
```diff
@@ -63,7 +63,7 @@ const SaleCreateView: React.FC<RouteComponentProps> = ({ location }) => {
 };
 
 const VoucherListView: React.FC<RouteComponentProps<{}>> = ({ location }) => {
-  const qs = parseQs(location.search.substr(1));
+  const qs = parseQs(location.search.substr(1), { arrayLimit: Infinity });
```
Comment (jwm0) [inline src/discounts/index.tsx:66]: Document why you need to do it this way, it looks confusing. Doesn't it also touch other parts of the app?

**Review by jwm0 — CHANGES_REQUESTED** [review]

**Comment by SektorDV** [comment]
turns out the problem was related to the way the application creates URL query strings (a number of places in the app used stringifyQs imported directly from the qs library rather than the one in utils). as a result, i removed the previous fix and globally replaced the stringifyQs imports across the app, which resolves the issue with vouchers and hopefully prevents similar issues from arising anywhere else. i checked the app for possible issues arising from the changes, but a doublecheck by QA probably couldn't hurt.

**Review by jwm0 — APPROVED** [review]
@SektorDV Please mention all the views that this change touches so QA can check each one of them

**Pull request merged**